terraform/stacks/umccr_data_portal/workflow/template/wgs_tumor_normal_stg.json

```diff
@@ -6,8 +6,10 @@
   "fastq_list_rows": [],
   "tumor_fastq_list_rows": [],
   "enable_map_align_output": true,
+  "enable_map_align_output_germline": false,
   "enable_duplicate_marking": true,
   "enable_sv": true,
+  "enable_sv_germline": false,
```

Comment:
So, just to ask/confirm; this two flags signal not to produce that extra BAM, correct?

Reply:
We set enable_map_align_output to false for dragen germline to not produce the extra bam.

enable_sv_germline needs to be set to false because enable_map_align_output needs to be set to be true in order for enable_sv to be set to be true.

Reply:
Thanks for explain. I will merge this, tick next. And, as for next action, please kindly help check in Trello card.